readthedocs/doc_builder/python_environments.py

```diff
@@ -318,8 +318,9 @@ class Conda(PythonEnvironment):
     .. _Conda: https://conda.io/docs/
     """
 
-    def venv_path(self):
-        return os.path.join(self.project.doc_path, 'conda', self.version.slug)
+    # pylint: disable=arguments-differ
```

Comment:
This also makes me worry this is not the right approach. It seems like having each subclass define this cleanly is a nicer?

Reply:
I don't have a strong preference here. It's the same code -- The only difference is the prefix (virtualenv or condaenv variable)

I put it in the super-class only to avoid repetition of the same logic.